data/hai/hai reveal 3 early.txt

```diff
@@ -0,0 +1,436 @@
+	stopover "Greenbloom"
+	stopover "Skyfarm"
+	stopover "Farwater"
+	stopover "Greenwater"
+	destination "Mountaintop"
+	clearance
+	passengers 1
+	blocked `Terry contacts you and says, "Captain, you're going to need a bunk free for me. Do you have space?" You do not, so you should complete or abort another mission to make room for her, then return here.`
+	on visit
+		dialog `You have arrived at the Mountaintop construction site but it isn't finished or staffed yet.`
+	to offer
+		has "Hai Building Mountaintop [1b] Return: done"
+	on offer
+		conversation
+			`At Hai-home you are directed to a now familiar government hangar.`
+			`	Sayari, Xilin, and Alondo all send you greeting pings, but are busy at the time of your landing. To your surprise though, Terry is banging on your airlock before too long.`
+			`	"Captain <last>, or should I call you <first>? Anyway, doesn't matter. How would you like to make some money?"`
+			`	The smile she gives you is something like you imagine a large cat giving the tasty morsel it just identified as its food.`
+			`	"Of course you do, everyone likes to make money," she says.`
+			`	"The Hai have put out the call for volunteers to staff the new Mountaintop station when it's finished. While you've been off watching early construction works, we've been gauging public interest and well... it's not going to work. In order to sell this they're going to need at least four thousand volunteers by the end of the week to get a good handle on what they can expect to need in terms of facilities, amenities, supplies, and various livability requirements. Nearly twelve thousand by the time the station is actually built. Our current estimates suggest they'll reach a quarter of that."`
+			`	As she's talking Terry has been laying out charts and graphs, and using a handheld Hai holographic device that she seems to have picked up somewhere to show animated diagrams.`
+			choice
+				`	"How do you know all this?"`
+			`	Terry raises an eyebrow at you, "You remember we have a supercomputer that could predict the economic situation across the whole of human habited space up to fifty years in the future to within 1% accuracy right?"`
+			`	When she puts it that way, it does suddenly feel like a bit of a stupid question.`
+			# And how did that turn out for you eh? lol
+			`	"As it turns out, that same computer is predicting that the Syndicate stands to lose billions of credits, per day, for the duration of any crisis evolving from this which involves the uncontrolled release of the Hai secret; however, we have a plan."`
+			choice
+				`	"And that is?"`
+				`	"What's the deal then?"`
+			`	Terry crosses her arms and leans back against the bulkhead, "We need a recruitment campaign, and if you haven't guessed, you're the man to do it."`
+			`	"The Syndicate will offer you a sum of 10,000,000 credits to visit every planet in Hai space and do glossies. I will stage manage, and have promotional material ready inside of 3 hours."`
```

Comment:
```suggestion
			`	Terry crosses her arms and leans back against the bulkhead, "We need a recruitment campaign, and if you haven't guessed, you're the man to do it. The Syndicate will offer you a sum of 10,000,000 credits to visit every planet in Hai space and do glossies. I will stage manage, and have promotional material ready inside of 3 hours."`
```
A little iffy on the flow of that last sentence. Grammatically, it's fine, but it just feels somewhat off. I'd suggest rephrasing, but I can't really think of a good way to do it.

Reply:
How about: "I'll obviously stage manage your image, and will have promotional material ready inside of 3 hours."

Reply:
Sounds good!